46.md

```diff
@@ -30,13 +30,13 @@ The client generates a QR code in the following form (URL-encoded):
 nostrconnect://<client-key-hex>?relay=wss://...&metadata={"name":"...", "url": "...", "description": "..."}
 ```
 
-The signer scans the QR code and sends a `connect` message to the client in the specified relays.
+The signer scans the QR code and sends a `connect` message to the client in the specified relays using the request event format.
```

Comment:
what is the request event format?

Reply:
oh, I see

maybe we should rephrase this to say "RPC call" somehow.

Reply:
Basically I'm just confused at how the signer can send a "connect" message to the client, when the signer's event format doesn't allow specifying a method.  It's a backwards command.  So I'm assuming it must use the other event format for this.  Given I don't actually know how this is supposed to be resolved I'm just not implementing this part of the spec for now.

Reply:
Ok I'm implementing 2. I commented out my impl of flow 1 since I couldn't understand how to send a connect from the server to the client.  I was hoping someone did that and could explain what has actually happened in practice.